.github/workflows/mavenLicenseCheck.yml

```diff
@@ -112,7 +112,7 @@ jobs:
 
     - name: Check license vetting status (and ask for review if requested)
       id: check-license-vetting
-      uses: eclipse/dash-licenses/.github/actions/maven-license-check-action@master
+      uses: eclipse/dash-licenses/.github/actions/maven-license-check-action@f12cb1ced8fb5d92acdd9a4ad5930075a94cc675 # master
```

Comment:
I would not use a fixed hash for this action.
It is an action hosted within the repo itself. When you change the workflow, you would have to change this workflow as well, referencing the new version of it (cant be done in the same commit btw). 

Also when someone is able to manipulate the tags of this repo, this should be our least concern.